app/src/main/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivity.kt

```diff
@@ -84,11 +85,11 @@ class AdministratorControlsActivity :
       supportFragmentManager.findFragmentById(
         R.id.administrator_controls_fragment_multipane_placeholder
       )
-    /**
-     * when fragment is [ProfileEditFragment] then we should switch
-     * fragment to [ProfileListFragment] or else
-     * we just end the activity as then we are
-     * having [ProfileListFragment] on top.
+    /*
+     when fragment is ProfileEditFragment then we should switch
+     fragment to ProfileListFragment or else
+     we just end the activity as then we are
+     having ProfileListFragment on top.
      */
```

Comment:
You should keep the '*' at the start of the lines, and you should wrap them at the 100 character mark.